port_ocean/core/handlers/entity_processor/jq_entity_processor.py

```diff
@@ -62,10 +61,15 @@ async def _search(self, data: dict[str, Any], pattern: str) -> Any:
 
     async def _search_as_bool(self, data: dict[str, Any], pattern: str) -> bool:
         loop = asyncio.get_event_loop()
+        start_time = loop.time()
         compiled_pattern = self._compile(pattern)
-        first_value_callable = functools.partial(compiled_pattern.first, data)
-        value = await loop.run_in_executor(None, first_value_callable)
-
+        func = compiled_pattern.input_value(data)
+        compile_time = loop.time() - start_time
+        value = await loop.run_in_executor(None, func.first)
+        execute_time = loop.time() - start_time - compile_time
+        logger.debug(
+            f"Search for pattern {execute_time:.2f} seconds, compile time {compile_time:.2f} seconds"
```

Comment:
This is golden, I wonder if it's possible to also send the explicit metrics as values in addition to the debug log text